de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,111 @@
+app-id: de.nagstamon.nagstamon
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: nagstamon
+rename-desktop-file: nagstamon.desktop
+rename-icon: nagstamon
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
```

Comment:
either it supports wayland and it is `fallback-x11` or it doesn't and the walyand socket should be removed.

Reply:
replaced now with fallback-x11

Reply:
Doesn't work natively with wayland so I removed wayland and readded x11.